webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/knossos/KnossosDataLayers.scala

```diff
@@ -27,7 +27,7 @@ trait KnossosLayer extends DataLayer {
   lazy val resolutions: List[Point3D] = sections.map(_.resolutions).reduce(_ union _).map {
     case Left(r) => Point3D(r, r, r)
     case Right(r) => r
-  }
+  }.toSet.toList
```

Comment:
What happens if one resolution uses `[1, 1, 1]` and another one uses the `1` shorthand? Will it still be de-duplicated?